pandas/core/series.py

```diff
@@ -111,21 +111,23 @@
 
 __all__ = ["Series"]
 
-_shared_doc_kwargs = {
-    "axes": "index",
-    "klass": "Series",
-    "axes_single_arg": "{0 or 'index'}",
-    "axis": """axis : {0 or 'index'}
-        Parameter needed for compatibility with DataFrame.""",
-    "inplace": """inplace : boolean, default False
-        If True, performs operation inplace and returns None.""",
-    "unique": "np.ndarray",
-    "duplicated": "Series",
-    "optional_by": "",
-    "optional_mapper": "",
-    "optional_labels": "",
-    "optional_axis": "",
-}
+_shared_doc_kwargs.update(
+    {
+        "axes": "index",
```

Comment:
only need to update those items which actually need updating otherwise this is very hard to read.